core/catalog-core-urlresourcereader/src/main/java/ddf/catalog/resource/impl/URLResourceReader.java

```diff
@@ -187,7 +184,6 @@ public ResourceResponse retrieveResource(URI resourceURI, Map<String, Serializab
 
         if (properties.containsKey(BYTES_TO_SKIP)) {
             bytesToSkip = properties.get(BYTES_TO_SKIP).toString();
-            logger.debug("bytesToSkip: {}", bytesToSkip);
```

Comment:
I agree with @jvettraino  at a minimum it seems like we would want to know the property was there. But it could be helpful knowing how many bytes were skipped unless that is redundantly being logged in multiple places. Could also make it at trace level.